src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetSpelling.java

```diff
@@ -143,7 +143,7 @@ public void onCancelled()
    @Override
    public void invalidateAllWords()
    {
-      lintManager_.forceRelint();
+      lintManager_.relintAfterDelay(-1);
```

Comment:
We should name this special value so it's clear what calling with `-1` does.

Reply:
👍